pkg/forward/forward.go

```diff
@@ -332,16 +332,20 @@ func (isdf *InterStepDataForward) forwardAChunk(ctx context.Context) {
 	// let's figure out which vertex to send the results to.
 	// update the toBuffer(s) with writeMessages.
 	for _, m := range udfResults {
-		// look for errors in udf processing, if we see even 1 error let's return. handling partial retrying is not worth ATM.
+		// look for errors in udf processing, if we see even 1 error flag the error boolean and
```

Comment:
```suggestion
		// look for errors in udf processing, if we see even 1 error 
```

didn't understand what the "flag the error boolean" meant.

Reply:
Ah sorry, typo here, was intended for a logic I removed later. Will remove